### PR TITLE
Print error log when neither left is green nor right is blue

### PR DESCRIPTION
**Pull request by shrekshao**
Print detailed pixels value error when neither left is green nor right is blue to help better understanding of flakiness.

### CODE REVIEW

File: sdk/tests/conformance/textures/misc/texture-video-transparent.html [hunk 1]
```diff
@@ -123,7 +123,7 @@
             testFailed("right is blue, but: " + redBlueError);
         }
     } else {
-        testFailed("neither left is green nor right is blue")
+        testFailed("neither left is green nor right is blue \n" + greenRedError + "\n" + redBlueError);
```
Comment (kenrussell) [inline sdk/tests/conformance/textures/misc/texture-video-transparent.html:126]: It seems to me that if the only error is that the left side isn't green, that this detail log won't show any more information.

**Review by kenrussell — APPROVED** [review]
Have you tried injecting various failures into this test (making each of the comparisons fail individually) and seeing what the error messages are? If not, please do and revise as necessary before merging. Thanks.

**Comment by kenrussell** [comment]
Note: I'd recommend printing out all of the read-back values from all of the locations very clearly in the situation where the test fails for any reason.

File: sdk/tests/conformance/textures/misc/texture-video-transparent.html [hunk 2]
```diff
@@ -92,21 +92,22 @@
     const tolerance = 5;
     const red = [255, 0, 0];
     const green = [0, 255, 0];
-    const blue = [0, 0, 255];
+    const blue = [0, 0, 200];
```
Comment (kenrussell) [inline sdk/tests/conformance/textures/misc/texture-video-transparent.html:95]: Did you mean to make this change, or was it only for testing?

Comment (shrekshao) [reply sdk/tests/conformance/textures/misc/texture-video-transparent.html:95]: that was meant to be deleted. BTW I'm extending to print the whole buf.

**Comment by shrekshao** [comment]
Now outputing whole buffer value when failed. Will merge and roll if that looks good. @kenrussell

File: sdk/tests/js/webgl-test-utils.js [hunk 3]
```diff
@@ -1283,7 +1283,7 @@ var checkCanvasRectColor = function(gl, x, y, width, height, color, opt_errorRan
           was += "," + buf[offset + j];
         }
         differentFn('at (' + (x + (i % width)) + ', ' + (y + Math.floor(i / width)) +
-                    ') expected: ' + color + ' was ' + was);
+                    ') expected: ' + color + ' was ' + was, buf);
```
Comment (kenrussell) [inline sdk/tests/js/webgl-test-utils.js:1286]: Hopefully this has no effect on other tests.

**Review by kenrussell — APPROVED** [review]
Looks good!

**Pull request merged**